### PR TITLE
For Create, match cmd log & cmd options

### PR DESCRIPTION
**Pull request by Manukapp**
For user's clarity, match wording for `--force` option  in `datalad create`. Currently, if error about non-empty dataset, cmd log suggests to use `force option`, **however**, the cmd log displays OK. Here I suggest FORCE.

**Command log:**
```
Running: Dataset('C:\\Users\\Leo\\Desktop\\Gooey\\Windy').create()

create(error): . (dataset) [will not create a dataset in a non-empty directory, use `--force` option to ignore]
```

### CODE REVIEW

File: datalad_gooey/simplified_api.py [hunk 1]
```diff
@@ -34,14 +34,14 @@
         ),
     ),
     create=dict(
-        name='C&reate a dataset',
+        name='Create a dataset',
```
Comment (mih) [inline datalad_gooey/simplified_api.py:37]: Please do not remove the keyboard navigation hint!

File: datalad_gooey/simplified_api.py [hunk 2]
```diff
@@ -34,14 +34,14 @@
...
         exclude_parameters=set((
             'initopts',
             'description',
             'fake_dates',
         )),
         parameter_display_names=dict(
-            force='OK if target directory not empty',
+            force='FORCE if target directory not empty',
```
Comment (mih) [inline datalad_gooey/simplified_api.py:44]: Makes sense. Maybe just `Force` (less dramatic capitalization)?

**Review by mih — CHANGES_REQUESTED** [review]
Good idea. Few changes suggested. thx!

File: datalad_gooey/simplified_api.py (hunk 1 above)
Comment (Manukapp) [reply datalad_gooey/simplified_api.py:37]: Sorry! thought that was a typo, didn't see it in the others haha..

**Comment by mih** [comment]
You did not push the actual change yet, right?

**Comment by Manukapp** [comment]
Hi, not sure how to do this, I updated the branch, and cloned my branch locally, but can't find how to push it, should I reopen another PR?

**Comment by adswa** [comment]
You can open another PR if that's easier for you, but if the necessary changes are on your local patch-3 branch, a git push origin patch-3 should do the job of updating this branch.

**Pull request closed without merging**